plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -39,19 +39,11 @@ import {
 import { PgInstrumentationConfig } from './types';
 import * as utils from './utils';
 import { AttributeNames } from './enums/AttributeNames';
-import {
-  SemanticAttributes,
-  DbSystemValues,
-} from '@opentelemetry/semantic-conventions';
 import { addSqlCommenterComment } from '@opentelemetry/sql-common';
 import { VERSION } from './version';
 
-const PG_POOL_COMPONENT = 'pg-pool';
-
 export class PgInstrumentation extends InstrumentationBase {
-  static readonly COMPONENT = 'pg';
-
-  static readonly BASE_SPAN_NAME = PgInstrumentation.COMPONENT + '.query';
+  static readonly BASE_SPAN_NAME = 'pg.query';
```

Comment:
for `pg.query` there is this `static readonly` on the class, but for other similar span names like `'pg.connect'` we have the value hardcoded in the `startSpan' call.
I would consider aligning the behavior to have consistency.

also, referencing this property from `utils.ts` creates a circular import (utils.ts depends on. iinstrumentation.ts and vice versa). it was already there previously, but since you touch it, might be a good chance to move somewhere else, like `consts.ts`.

Reply:
Done, I've tried to match the style with enums that already was in place